src/test/java/seedu/address/logic/parser/WishfulShrinkingParserTest.java

```diff
@@ -125,14 +129,25 @@ public void parseCommand_selectRecipe() throws Exception {
         assertEquals(new SelectRecipeCommand(INDEX_FIRST_CONSUMPTION), command);
     }
 
-    //@Test
-    //public void parseCommand_edit() throws Exception {
-    //    Recipe recipe = new RecipeBuilder().build();
-    //    EditRecipeCommand.EditRecipeDescriptor descriptor = new EditRecipeDescriptorBuilder(recipe).build();
-    //    EditRecipeCommand command = (EditRecipeCommand) parser.parseCommand(EditRecipeCommand.COMMAND_WORD + " "
-    //            + INDEX_FIRST_RECIPE.getOneBased() + " " + RecipeUtil.getEditRecipeDescriptorDetails(descriptor));
-    //    assertEquals(new EditRecipeCommand(INDEX_FIRST_RECIPE, descriptor), command);
-    //}
+//    @Test
+//    public void parseCommand_editRecipe() throws Exception {
```

Comment:
dont forget to remove this

Reply:
this test im waiting for jq's stuff which will fix it